content/synthetics/uptime_check.md

```diff
@@ -0,0 +1,88 @@
+Uptime Checks are HTTP requests (GET, POST, PUT, etc.) executed by Datadog to your web properties or application endpoints at configurable periodic intervals from multiple locations around the world. These Checks verify not only that your applications are up and responding to requests, but that any conditions you define are met, such as response time, HTTP response code, and header or body contents.
+
+## Configuration
+### Request
+
+Define the request you want to be executed by Datadog:
+
+{{< img src="synthetics/uptime_check/uptime_check_make_request.png" alt="Uptime Check make request" responsive="true" style="width:80%;">}}
+
+1. Choose the `Method` and `URL` to query. Available `Method` are:
+    * GET
+    * POST
+    * PATCH
+    * PUT
+    * DELETE
+    * OPTIONS
+
+2. Optionally, click on **Advanced options** to enrich your request with custom Request Headers, Authentication credentials, body content, or cookies.
+
+3. Name your Uptime Check.
+4. Optionally, add tags to filter your Uptime Check in the [Synthetics page][1].
+
+5. Pick-up locations to run the test from, available locations are:
+    * Frankfurt (Request made from an AWS Datacenter)
+    * Tokyo (Request made from an AWS Datacenter)
+
+6. Choose a Check frequency between "1 run per minute" to "1 run per week".
+7. Finish by clicking on **Test URL** to try out the request configuration. You should see a response preview show up in the right side of your screen.
+
+### Validation
+
+When running an Uptime Check, you must define at least one Assertions that should be monitored by Datadog. An Assertion is defined by a parameter, a comparator, and a value.
+
+| Parameter     | Comparators                | Value type                         |
+| --------      | ---------                  | --------                           |
+| Status Code   | `is`, `is not`             | Integer                            |
+| Response time | `less than`                | Integer                            |
+| Headers       | `is`, `is not`, `contains` | Strings                            |
+| body          | `contains`                 | Strings ([regex are supported][4]) |
+
+If you define an assertion on the content of `Headers`, you must define the headers name and the associated value.
+
+Click on **Add new assertion** to add up to ten assertions for your Uptime Check. 
+
+**Note**: If you clicked on **Test URL**, then the basic Assertions are automatically filled:
+
+{{< img src="synthetics/uptime_check/assertion.png" alt="Assertions" responsive="true" style="width:80%;">}}
+
+Use the **Test URL** response to quickly add headers from the test response as one of the Uptime Check assertions:
+
+{{< img src="synthetics/uptime_check/assertion_setup.gif" alt="Assertions" responsive="true" style="width:80%;">}}
+
+### Notifications
+
+A notification is sent if at least one of the Assertion fails. To configure notifications:
+
+1. Select users and/or [services][2] to send the notifications to.
```

Comment:
Not sure but it may worth to explain that this step is just to help putting the users / services into the message below, but the recipients / message can still be edited manually?